Desktop/components/JASP/Controls/Form.qml

```diff
@@ -184,8 +183,8 @@ AnalysisForm
 		running:		false
 		repeat:			false
 		interval:		0
-		onTriggered:	formCompleted();
+		onTriggered:	formCompletedSignal();
 	}
-	
-	Component.onCompleted:	bindingTimer.start()
+
+	Component.onCompleted:	bindingTimer.start() //Do we really need both a timer and a Queued connection?
```

Comment:
@boutinb no right?

Reply:
Maybe I do a cleanup round btw, I see some things I might as well remove entirely.
Today or tomorrow.

Reply:
or tomorrow, with the other changes...